### ISSUE TITLE
[7.12.1] Core Crashed Error

### ISSUE THREAD

**Issue by drew2a** [Tribler/tribler#7034]
```
tribler.gui.utilities.CreationTraceback: , 
  File "run_tribler.py", line 100, in <module>, 
  File "tribler\gui\start_gui.py", line 77, in run_gui, 
  File "tribler\gui\utilities.py", line 410, in trackback_wrapper, 
  File "tribler\gui\core_manager.py", line 101, in start_tribler_core, 
The above exception was the direct cause of the following exception:, 
Traceback (most recent call last):, 
  File "tribler\gui\utilities.py", line 413, in trackback_wrapper, 
  File "tribler\gui\utilities.py", line 410, in trackback_wrapper, 
  File "tribler\gui\core_manager.py", line 187, in on_core_finished, 
tribler.gui.exceptions.CoreCrashedError: The Tribler core has unexpectedly finished with exit code 0 and status: 0!, 
Last core output: , 
 Sentry is attempting to send 1 pending error messages, 
Waiting up to 2 seconds, 
Press Ctrl-Break to quit, 
Sentry is attempting to send 1 pending error messages, 
Waiting up to 2 seconds, 
Press Ctrl-Break to quit
```
17+ users affected

**Comment by drew2a** [comment]
Still occurs in `7.12.1`

**Comment by KAAtheWiseGit** [comment]
I have encountered a similar issue on Ubuntu 22.04. Tribler crashes every time I try to open the native file selection window. The crash-log points to `trackback_wrapper`:
```
  File "tribler/gui/utilities.py", line 411 in trackback_wrapper
  File "tribler/gui/start_gui.py", line 77 in run_gui
  File "run_tribler.py", line 100 in <module>
```
![Screenshot from 2022-11-15 13-24-59](https://user-images.githubusercontent.com/97791001/201903060-ad42c1fa-f3ca-48b0-9b7c-404fbf022c0d.png)
```
  INFO - tribler.core.components.gui_process_watcher.gui_process_watcher(43) - GUI process is not working, initiate Core shutdown
  INFO - tribler.core.start_core(110) - Shutdown event fired
```

**Comment by drew2a** [comment]
@KAAtheWiseGit 
>  Tribler crashes every time I try to open the native file selection window.

Do I understand correctly that you have this issue reproducible with the current steps?
 1. Open the Tribler
 2. Search for a torrent
 3. Click on the torrent
 4. Click on the download button 
 5. Tribler crashes

**Comment by KAAtheWiseGit** [comment]
@drew2a

I have uninstalled Ubuntu, so I cannot reproduce the crash now.

But back then the crash occurred without me downloading any torrents. The app crashed when I tried to upload a torrent from file and when I tried to change the destination folder for Tribler downloads. Both actions were supposed to open the native file manager and let me choose a file using it.

**Comment by drew2a** [comment]
@KAAtheWiseGit thank you for the clarification!

**Comment by kozlovsky** [comment]
Should be fixed in #7349
